k8up/values.yaml

```diff
@@ -1,7 +1,21 @@
 ---
-k8up_operator:
-  image: docker.io/vshn/k8up:v0.1.6
+
+image:
+  pullPolicy: IfNotPresent
+  repository: docker.io/vshn/k8up
+  tag: v0.1.6
+
+imagePullSecrets: []
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name:
+
+k8up:
   ## Allows the specification of additional environment variables
+  extraEnvVars: []
```

Comment:
What's the point of the `extraEnvVars` ?

Reply:
I dunno. Was there before. The only reason I can imagine is that we can leave extraEnv with `BACKUP_IMAGE` variable unmodified if one wants to add more variables. But if `BACKUP_IMAGE` is required anyway, we might as well include that fixed in the deployment and provide an own helm variable just for the backup image. WDYT @Kidswiss ?

Reply:
A lot of configuration can be passed to the operator via the envvars. But the only one that's mandatory is the image version, so it makes sense to add this as a helm variable.